src/renderer/scss/_gui.scss

```diff
@@ -34,6 +34,11 @@ body
   color: var(--color-meta-light);
 }
 
+.credit-amount--bold
```

Comment:
is this used elsewhere in the app or is it just for shapeshift? I don't have a criticism here, I'm just wondering what we decided about using reusable vs case specific classes?